File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    answer = ''
    ref1 = "abcdefghijklmnopqrstuvwxyz"
    ref2 = ref1.upper()
    
    for item in s:
        if item in ref1:
            index = ref1.index(item)
            answer += ref1[(index + n) % 26]
        elif item in ref2:
            index = ref2.index(item)
            answer += ref2[(index + n) % 26]
        else:
            answer += " "
            
    return answer

File: 프로그래머스/1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    let answer = '';
    const lowerAlphabet = "abcdefghijklmnopqrstuvwxyz";
    const upperAlphabet = lowerAlphabet.toUpperCase();
    for (const item of s) {
        if (lowerAlphabet.includes(item)) {
            answer += lowerAlphabet[(lowerAlphabet.indexOf(item) + n) % 26];
        } else if (upperAlphabet.includes(item)) {
            answer += upperAlphabet[(upperAlphabet.indexOf(item) + n) % 26];
        } else {
            answer += " ";
        }
    }
    return answer;
}